utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -114,9 +114,9 @@
 "CDDL or GPLv2 with exceptions": CDDL-1.0 OR GPL-2.0-only WITH Classpath-exception-2.0
 "CDDL v1.0 / GPL v2 dual license": CDDL-1.0 OR GPL-2.0-only
 "CDDL v1.1 / GPL v2 dual license": CDDL-1.0 OR GPL-2.0-only
-"CDDL+GPL License": CDDL-1.0 AND GPL-2.0-or-later
-"CDDL+GPL": CDDL-1.0 AND GPL-2.0-or-later
-"CDDL+GPLv2": CDDL-1.0 AND GPL-2.0-only
+"CDDL+GPL License": CDDL-1.0 OR GPL-2.0-or-later
```

Comment:
Commit message nits: "a**n** 'AND'", "a**n** 'OR'", "licenses **are** incompatible"

Reply:
Still says "a 'OR'"

Reply:
Fix it now